core/retries/src/main/java/software/amazon/awssdk/retries/internal/RetryStrategyDefaults.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retries.internal;
+
+import software.amazon.awssdk.annotations.SdkProtectedApi;
+import software.amazon.awssdk.retries.api.RetryStrategy;
+
+/**
+ * The set of retry predicates that are by default added to a retry strategy.
+ */
+@SdkProtectedApi
+public interface RetryStrategyDefaults {
+
+    /**
+     * @return The unique name that identifies this set of predicates
+     */
+    String name();
```

Comment:
If we implement `equals()` and `hashCode()`, we don't even need `name()`.

Reply:
That or just use identity equality and singletons, but I suppose that names are less opaque.

Reply:
I think I slightly prefer singleton and identity, lets got with that. We wouldn't need to implement hashCode and  equals then.

Reply:
changed my mind, lets just keep `name()`,  it's simpler